Proposal for Structural Changes

akeller opened the pull request:
Created the following pages in reference:
* Licenses
* Notices
* Supported environments

Moved and modified: 
* Release cycle (from open-source/zeebe to reference) _-> Renamed **Release policy** to align with Camunda Platform_
* Deprecated features _-> Renamed **Announcements** to align with Camunda Platform_

Modified:
* Contact

Added to footer:
* Licenses
* Notices
* Release policy (formerly Release cycle)
* Clickable logo to Camunda.com

**Open questions**

- [x] Is there any other license information that we need to pull in? 
- [x] I assumed there were no security notices for Camunda Cloud yet, is that correct?
- [x] @menski can you update the release cycle with information relevant for Camunda Cloud? ~~I modified it slightly, but the section for Zeebe 1.0 should be rewritten since we know there are breaking changes.~~

felix-mueller commented on docs/reference/notices.md:
```diff
@@ -0,0 +1,9 @@
+id: notices
+title: "Notices"
+---
+### Security Notices
```
What about general announcements? We do this for Camunda platform: https://docs.camunda.org/enterprise/announcement/

akeller replied:
I'm a bit torn about how to address this. It looks like half the content is announcing new supported features and frameworks and the other half is deprecation announcements. Is that how it will work for Camunda Cloud? 

I'd like to move https://docs.camunda.io/docs/product-manuals/zeebe/open-source/deprecated-features out of Zeebe specifically and rename the page from "deprecated features" to "announcements". Do we have any new items we need to list for GA?

felix-mueller replied:
I assume for Cloud we will handle it in a similar way for now. 
For 1.0 topics @menski  probably knows best what environments we do not support anymore. Right now Elastic 6 comes to my mind. Elastic 7 is the new thing

akeller replied:
@menski Do we need to include Elastic 6 as an announcement for removing support? Or is your migration guide moving from 0.26 to 1.0 going to cover this?

felix-mueller replied:
For Camunda platform (including Optimize) we are announcing it every time we do not support a specific database anymore. So if 6 is not supported anymore I suggest we announce it.

akeller replied:
@felix-mueller I'm going to resolve these comments and start a new thread. I have an idea 💡

akeller commented on src/pages/contact.md:
```diff
@@ -7,10 +7,12 @@ description: Contact Camunda
+- We also accept **feature requests** through a similar process. Please open an issue in this [repository](https://github.com/camunda-cloud/issues). The feature requests have a default template, so please describe what you'd like to see in detail.
 
-- **Feebdack and Support** can be submitted or requested via the corresponding entry in the [Cloud Console navigation menu](../../docs/product-manuals/cloud-console/troubleshooting/feedback-and-support). If you have a **license agreement** with us you will be redirected to the [Support queue](https://jira.camunda.com/projects/SUPPORT/) at Camunda. In the Free Plan, contact can be made using the i ...
+- For **security-related issues**, please e-mail us at [this address](mailto:cloud-security@camunda.com) so we can solve the problem as quickly as possible. Do not use GitHub for security-related issues.
```
@felix-mueller @menski please update the Support and Security process, links, and contact info.

akeller replied:
These are being worked in a Google doc, I'll take care of this as we solidify the processes internally.

akeller commented:
@felix-mueller take a look at my last commit - https://github.com/camunda-cloud/camunda-cloud-documentation/pull/210/commits/6d57fadb4e1ee7ed04b76cc3e06cf3f90269f037

I renamed Deprecated Features to Announcements to better align with Camunda Platform. I also moved the supported environments outside of the Release Cycle (now Release Policy, again to align with Camunda Platform) and moved it into the Announcements page.

This means the Announcements page now handles supported & unsupported environments and deprecated features.

When did we deprecate or stop supporting Elastic 6? At a particular version or should we just list it as an Unsupported environment?  Can you and @menski review the existing supported environment content and correct it if necessary?

felix-mueller commented on docs/reference/release-policy.md:
```diff
@@ -0,0 +1,19 @@
+- A new minor version of the Camunda Cloud components every three months
+- In between minor versions, two alpha releases (to preview the upcoming minor version)
+
+At the time of writing, Zeebe supports the last two released minor versions with
```
Maybe we should replace "Zeebe" With "Camunda Cloud"?

felix-mueller commented on docs/reference/licenses.md:
```diff
@@ -0,0 +1,15 @@
+
+Licenses and license information for Zeebe can be found on the Zeebe project [README](https://github.com/camunda-cloud/zeebe#license).
+
+#### Camunda Cloud Documentation
```
does it make sense to link here too? https://camunda.com/legal/terms/cloud-terms-and-conditions/ ?

akeller replied:
Good idea, I will add 🙂

menski commented on docs/reference/notices.md:
```diff
@@ -0,0 +1,18 @@
+---
+id: notices
+title: "Notices"
```
Should we call this page "Security Notices", otherwise I'm not sure I would find where to report a security vulnerability. Or is "Notices" imply security and I'm just not aware of it?

akeller replied:
This is the title in the sidebar on docs.camunda.org, but looking closer I realize contextually it sits in the "Security" section. I will adjust the name.

menski commented on docs/reference/overview.md:
```diff
@@ -7,10 +7,20 @@ slug: /reference/
+## Security, support, & license information
+- [Announcements](announcements.md)
+- [Licenses](licenses.md)
+- [Notices](notices.md)
```
Similar to my comment on the notices page, in the heading we are listing "Security", but then non of the sub-pages contains the word security and I'm not sure which page to visit. Please ignore again if I'm just not aware that "notices" is commonly understood term for security reporting.

akeller replied:
Took care of this, "Notices" will now be "Security Notices".

menski commented on docs/reference/supported-environments.md:
```diff
@@ -0,0 +1,21 @@
+## Zeebe
+
+- **Zeebe Broker/Gateway** - the cluster components of Zeebe require OpenJDK 11+
+  and optional if the Elasticsearch exporter is used Elasticsearch 6.8.x
```
The elastic search version is now `7.12.x`

```suggestion
  and optional if the Elasticsearch exporter is used Elasticsearch 7.12.x
```

menski commented on docs/reference/supported-environments.md:
```diff
@@ -0,0 +1,21 @@
+## Camunda Operate
+
+- **Operate Web App/Importer/Archiver** - the server components of Camunda
+  Operate require OpenJDK 11+ and Elasticsearch 6.8.x
```
```suggestion
  Operate require OpenJDK 11+ and Elasticsearch 7.12.x
```

menski commented on src/pages/contact.md:
```diff
@@ -7,10 +7,14 @@ description: Contact Camunda
+- We also accept **feature requests** through a similar process. Please open an issue in this [repository](https://github.com/camunda-cloud/issues). The feature requests have a default template, so please describe what you'd like to see in detail.
 
-- **Feebdack and Support** can be submitted or requested via the corresponding entry in the [Cloud Console navigation menu](../../docs/product-manuals/cloud-console/troubleshooting/feedback-and-support). If you have a **license agreement** with us you will be redirected to the [Support queue](https://jira.camunda.com/projects/SUPPORT/) at Camunda. In the Free Plan, contact can be made using the i ...
+- For **security-related issues**, please e-mail us at [this address](mailto:cloud-security@camunda.com) so we can solve the problem as quickly as possible. Do not use GitHub for security-related issues.
```
This should now link to the notices page or?

akeller replied:
Yes good catch! Thank you!

menski commented on src/pages/contact.md:
```diff
@@ -7,10 +7,14 @@ description: Contact Camunda
-- **Feebdack and Support** can be submitted or requested via the corresponding entry in the [Cloud Console navigation menu](../../docs/product-manuals/cloud-console/troubleshooting/feedback-and-support). If you have a **license agreement** with us you will be redirected to the [Support queue](https://jira.camunda.com/projects/SUPPORT/) at Camunda. In the Free Plan, contact can be made using the i ...
+- For **security-related issues**, please e-mail us at [this address](mailto:cloud-security@camunda.com) so we can solve the problem as quickly as possible. Do not use GitHub for security-related issues.
+
+- **Feedback and Support** can be submitted or requested via the corresponding entry in the [Cloud Console navigation menu](../../docs/product-manuals/cloud-console/troubleshooting/feedback-and-support). 
```
I guess we can save the user some time, the feedback and support button leads to forum for trail users, and JIRA for the paid users. Why aren't we linking directly to the cloud forum here, and then add half-sentence like "if you are an enterprise customer, please see the enterprise documentation" with a link to the enterprise docs. I guess we still have to create the enterprise docs?

Also a second thought, pointing people to camunda cloud console is strange for self managed and cloud enterprise on-prem users, which might not have an account in SaaS.

akeller replied:
> I guess we still have to create the enterprise docs?

That's the exact question I was asking myself. 

For launch, should we clearly mentioned 2 paths for support, forum and JIRA? The JIRA path is creating a ticket in the Support project, right?

menski replied:
Yes the support (paid customer) path is to create an Jira issues in the Support project.

akeller replied:
- **Feedback and Support** can be submitted or requested via JIRA for subscription or enterprise customers in the Support project. Otherwise, please use the community forum. For more information about Enterprise support and additional support resources please see [Enterprise Support](https://camunda.com/support/).

^ Would something like that work for now?

menski replied:
yes sounds good to me

menski reviewed (commented):
I like the changes just, some smaller comments, nothing which should block us from making progress

akeller commented on docs/guides/introduction-to-camunda-cloud.md:
```diff
@@ -7,4 +7,13 @@ slug: /guides/
+
+* Zeebe - Zeebe is the cloud native process engine of Camunda Cloud.
+* Operate - Manage, monitor, and troubleshoot your processes through Operate.
+* Tasklist - Use Tasklist to customize how people complete assigned tasks.
```
I like your suggestion, I will make this change. This language was pulled from a product marketing presentation.
